anta/tests/snmp.py

```diff
@@ -339,3 +340,74 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(f"The following SNMP error counters are not found or have non-zero error counters:\n{error_counters_not_ok}")
+
+
+class VerifySnmpLogging(AntaTest):
+    """Verifies SNMP logging and SNMP manager(host) details.
+
+    This test performs the following checks for each specified host:
+
+     1. Verifies that the SNMP logging is enabled on the device.
+     2. Verifies SNMP host matches the expected value.
+     3. Ensures that VRF provided matches the expected value.
```

Comment:
SNMP logging is checked once in the logic, not for each host.
```suggestion
    This test performs the following checks:

     1. SNMP logging is enabled globally.
     2. For each specified SNMP host:
         - Host exists in configuration.
         - Host's VRF assignment matches expected value.
```